release/charts/charts.go

```diff
@@ -0,0 +1,58 @@
+package charts
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+
+	ecmConfig "github.com/rancher/ecm-distro-tools/cmd/release/config"
```

Comment:
Does this need the import alias?

Reply:
To be honest I copied it from: 
`release/k3s/k3s.go` but I can remove it.
I just wanted to follow the pattern that was already there from the example I caught.

Reply:
Let's avoid import aliases if the imported package name doesn't conflict with any other imported or existing symbol.